src/index.scss

```diff
@@ -15,6 +12,12 @@ body {
 #root {
   min-height: 100vh;
 }
+div {
```

Comment:
strange styles, usually the default typography styles are applied to the body element